SocialCareCaseViewerApi/V1/Factories/ResponseFactory.cs

```diff
@@ -117,5 +117,14 @@ public static WorkerResponse ToResponse(this Domain.Worker worker)
                 Teams = worker.Teams
             };
         }
+
+        private static string FormatFormNameForHistoricCaseNote(string noteType)
+        {
+            string pattern = @"\([^()]*\)$"; // Match brackets at the end e.g. (ASC)
+            var formName = noteType ?? "Case note";
```

Comment:
We might want to use `String.IsNullOrEmpty` here instead of nullish coalescing just in case there are any empty strings ("")

Reply:
That makes sense, will do dat!

Reply:
Updated in https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/202/commits/0311598b3a21b7c440b20d8a7232897dc295f77f.